backend/src/main/java/com/woowacourse/friendogly/club/domain/Club.java

```diff
@@ -188,34 +197,17 @@ public boolean isEmpty() {
         return clubMembers.isEmpty();
     }
 
-    public void removeClubMember(Member member) {
-        ClubMember target = clubMembers.stream()
-                .filter(e -> e.getMember().getId().equals(member.getId()))
-                .findAny()
-                .orElseThrow(() -> new FriendoglyException("참여 중인 모임이 아닙니다."));
-        clubMembers.remove(target);
-        if (isOwner(target)) {
-            updateOwner();
-        }
-        target.updateClub(null);
-        removeClubPets(member);
-    }
-
-    private void updateOwner() {
-        if (!isEmpty()) {
-            this.owner = clubMembers.get(0).getMember();
-        }
+    public boolean isOwner(ClubMember target) {
+        return findOwner().isSameMember(target.getClubMemberPk().getMember());
     }
 
-    private boolean isOwner(ClubMember target) {
-        return owner.getId().equals(target.getMember().getId());
+    public Name findOwnerName() {
+        return findOwner().getClubMemberPk().getMember().getName();
```

Comment:
```suggestion
    public String findOwnerName() {
        return findOwner().getClubMemberPk().getMember().getName().getValue();
```

밖에서 값을 쓰려면 결국 매번 `getValue()`를 호출해야겠네요